packages/ref/tests/unit/test_config.py

```diff
@@ -127,13 +127,15 @@ def test_defaults(self, monkeypatch):
         without_defaults = cfg.dump(defaults=False)
 
         assert without_defaults == {
+            "log_level": "INFO",
```

Comment:
I have no idea why cattrs thinks this is a default value.

We don't use the "no-defaults" output so it isn't a major